data/transit/route/subway.json

```diff
@@ -355,19 +363,15 @@
       }
     },
     {
-      "displayName": "VRR",
+      "displayName": "Verkehrsverbund Rhein-Ruhr",
       "id": "vrr-447f34",
       "locationSet": {"include": ["de"]},
-      "matchNames": [
-        "verkehrsverbund rhein-ruhr"
-      ],
+      "matchNames": ["vrr"],
       "tags": {
-        "network": "VRR",
+        "network": "Verkehrsverbund Rhein-Ruhr",
+        "network:short": "VRR",
         "network:wikidata": "Q448199",
         "network:wikipedia": "de:Verkehrsverbund Rhein-Ruhr",
-        "operator": "VRR",
-        "operator:wikidata": "Q448199",
-        "operator:wikipedia": "de:Verkehrsverbund Rhein-Ruhr",
```

Comment:
Is the actual operator known for this subway?

Reply:
There are several operators within the network: BOGESTRA, Ruhrbahn, Rheinbahn, DVG. For more details see:
https://ptna.openstreetmap.de/results/DE/NW/DE-NW-VRR-Analysis.html#A2.3.14

Reply:
Ah okay